pkg/sql/logictest/testdata/logic_test/redact_descriptor

```diff
@@ -0,0 +1,147 @@
+                "type": {
+                    "family": "IntFamily",
+                    "oid": 20,
+                    "width": 64
+                }
+            }
+        ],
+        "createAsOfTime": {},
+        "families": [
+            {
+                "columnIds": [
+                    1,
+                    2
+                ],
+                "columnNames": [
+                    "i",
+                    "j"
+                ],
+                "defaultColumnId": 2,
+                "name": "primary"
+            }
+        ],
+        "formatVersion": 3,
+        "id": 106,
+        "indexes": [
+            {
+                "createdAtNanos": "1662996773119411000",
+                "foreignKey": {},
+                "geoConfig": {},
+                "id": 2,
+                "interleave": {},
+                "keyColumnDirections": [
+                    "ASC"
+                ],
+                "keyColumnIds": [
+                    2
+                ],
+                "keyColumnNames": [
+                    "j"
+                ],
+                "keySuffixColumnIds": [
+                    1
+                ],
+                "name": "foo_j_idx",
+                "partitioning": {},
+                "predicate": "i = _:::INT8",
+                "sharded": {},
+                "version": 3
+            }
+        ],
+        "modificationTime": {},
+        "name": "foo",
+        "nextColumnId": 3,
+        "nextConstraintId": 2,
+        "nextFamilyId": 1,
+        "nextIndexId": 3,
+        "nextMutationId": 1,
+        "parentId": 104,
+        "primaryIndex": {
+            "constraintId": 1,
+            "createdAtNanos": "1662996773119411000",
+            "encodingType": 1,
+            "foreignKey": {},
+            "geoConfig": {},
+            "id": 1,
+            "interleave": {},
+            "keyColumnDirections": [
+                "ASC"
+            ],
+            "keyColumnIds": [
+                1
+            ],
+            "keyColumnNames": [
+                "i"
+            ],
+            "name": "foo_pkey",
+            "partitioning": {},
+            "sharded": {},
+            "storeColumnIds": [
+                2
+            ],
+            "storeColumnNames": [
+                "j"
+            ],
+            "unique": true,
+            "version": 4
+        },
+        "privileges": {
+            "ownerProto": "root",
+            "users": [
+                {
+                    "privileges": 2,
+                    "userProto": "admin",
+                    "withGrantOption": 2
+                },
+                {
+                    "privileges": 2,
+                    "userProto": "root",
+                    "withGrantOption": 2
+                }
+            ],
+            "version": 2
+        },
+        "replacementOf": {
+            "time": {}
+        },
+        "unexposedParentSchemaId": 105,
+        "version": "1"
+    }
+}
```

Comment:
Do you want to throw in a view and a UDT and a CTAS in there as well? We'll have to at some point anyway.